chart/etcd-backup-restore/templates/etcd-statefulset.yaml

```diff
@@ -200,6 +200,12 @@ spec:
             secretKeyRef:
               name: {{ .Release.Name }}-etcd-backup
               key: "storageAPIEndpoint"
+  {{- if .Values.backup.gcs.enableGCSEmulator }}
```

Comment:
An `{{- end }}` statement which is supposed to end the `{{- if .Values.backup.gcs.storageAPIEndpoint }}` block is missing after line 202 - right after `key: "storageAPIEndpoint"` which is cause for the lint error.

Please add this to fix the lint error and thereby fix the build.

Reply:
My bad. The build was showing the error after line 203 and I had no clue what went wrong there. 
Thanks a lot @renormalize for finding this.